build/specifications/GitVersion.json

```diff
@@ -76,9 +76,10 @@
           },
           {
             "name": "UpdateAssemblyInfoFileName",
```

Comment:
UpdateAssemblyInfoFileNameS, plural

Reply:
I've updated the PR according to your comment